src/lib.rs

```diff
@@ -229,6 +229,11 @@ pub trait Time {
 // Check that the Time trait is object-safe
 const _: Option<&dyn Time> = None;
 
+/// Format string used by Ruby [`Time#asctime`] method.
+///
+/// [`Time#asctime`]: <https://ruby-doc.org/core-3.1.2/Time.html#method-i-asctime>
```

Comment:
do you think it's worthwhile to add an `# Examples` section using `strftime::buffered`?

Reply:
I don't think it is strictly necessary, since we already link to the Ruby doc where an example is clearly visible. This is also a single specifier, so the table in the crate documentation already contains an example.

Reply:
Cool sounds good